scripts/remote-heal/remote-healer.py

```diff
@@ -165,6 +166,14 @@ def main(self):
             # Run reporting to quiet down trigger
             cmd = self.ossh_cmd(self._args.host,
                                 'docker exec oso-rhe7-host-monitoring /usr/bin/cron-send-ovs-stats')
+        elif re.search(r'^\[HEAL\] Critically High Memory usage of  docker  on', self._args.trigger):
+            logging.info("Restarting docker on " + self._args.host)
+
+            #run playbook to evacuate the host and restart the docker
+            #haven't found where should the script be ,maybe /root ?
```

Comment:
don't we know where the script will be (ie /usr/bin/heal_for_docker_use_too_much_memory.yml)?